smallrye-reactive-messaging-provider/src/main/java/io/smallrye/reactive/messaging/AbstractMediator.java

```diff
@@ -65,13 +117,48 @@ public void initialize(Object bean) {
                     try {
                         return this.configuration.getMethod().invoke(bean, args);
                     } catch (Exception e) {
+                        e.printStackTrace();
                         throw ex.processingException(configuration.methodAsString(), e);
                     }
                 };
             }
         }
     }
 
+    @SuppressWarnings("unchecked")
+    protected <T> T invoke(Message<?> message) {
+        try {
+            Objects.requireNonNull(this.invoker, msg.invokerNotInitialized());
+            Object[] objects = invocationParameters.apply(message);
+            return (T) this.invoker.invoke(objects);
```

Comment:
Do we need to store `objects` or can result be passed directly into call below?

Reply:
Done.